calicoctl/commands/node/run.go

```diff
@@ -141,6 +142,10 @@ Options:
                            within the Docker containers that have been networked
                            by the Calico driver.
                            [default: cali]
+     --enable-docker-networking-labeling
+                           Enables adding labels to Docker containers that can
+                           be used in policies. Setting this flag also disables
+                           the default network profiles.
```

Comment:
Actually, given that this flag enables one feature and disables another, I wonder if it's named correctly.

I'm mulling over whether something like    `--use-docker-networking-label-policy` would be more descriptive.  Not sure.  WDYT?